recipe/meta.yaml

```diff
@@ -15,6 +15,9 @@ build:
     - isympy = isympy:main
 
 requirements:
+  build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
```

Comment:
Is target_platform still undocumented?

Reply:
yes

Reply:
Probably isn't safe to use then

Reply:
This variable is used in hundreds of feedstocks at this point, in anaconda's own recipes, and in smithy. Anaconda unilaterally dropping API support for it would be hugely destructive. So despite it being not documented, it is perfectly safe to use I think.